### PR TITLE
remove docker from runlevel since it is redundant

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,12 +15,10 @@ ENV TF_IN_AUTOMATION="yes"
...
 # Install dependencies
-RUN apk add --update docker bash git openrc openssh openssl python3
-
+#
 # We run Docker as Docker-out-of-Docker (DooD). DooD is a solution where you run the Docker CLI inside a container,
-# and connect it to the host’s Docker by virtue of mount the /var/run/docker.sock into the container. Docker should be
-# started on container startup.
-RUN rc-update add docker boot
+# and connect it to the host’s Docker by virtue of mount the /var/run/docker.sock into the container.
+RUN apk add --update docker bash git openrc openssh openssl python3
```
Comment (mariux) [inline Dockerfile:21]: why do we need to add docker at all?

Comment (soerenmartius) [reply Dockerfile:21]: we basically only need the cli

Comment (mariux) [reply Dockerfile:21]: to do what?

Comment (soerenmartius) [reply Dockerfile:21]: We are using "docker out of docker" in order to run the forensics https://github.com/mineiros-io/build-tools/blob/master/Makefile#L84

Comment (mariux) [reply Dockerfile:21]: yes, but this is not in the buil-tools container... this a a snyk container running a test on the container...  afaiu?

Comment (soerenmartius) [reply Dockerfile:21]: yep but it's run inside the build tools container

Comment (mariux) [reply Dockerfile:21]: Where do we run it there?

Comment (soerenmartius) [reply Dockerfile:21]: We do not only use it for snyk but also for some tests, e.G. https://github.com/mineiros-io/terraform-aws-ecr/blob/master/test/ecr_repository_test.go#L99

This is also where in it crucial afaik. Actually, Snyk runs DooD inside its own container that isn't run by build-tools.

Comment (soerenmartius) [reply Dockerfile:21]: As you can see, for the tests it is mandatory.

Comment (soerenmartius) [reply Dockerfile:21]: Imho, a good solution would be to install the CLI only.

Comment (mariux) [reply Dockerfile:21]: Is the ecr thing running the Client or using the api directly?

Comment (mariux) [reply Dockerfile:21]: approved ... we shoudl check how to only install docker cli and if the docker go code actually needs the binary..?

Comment (soerenmartius) [reply Dockerfile:21]: Let me look into that before we merge this.

Comment (mariux) [reply Dockerfile:21]: too late ;) ... can be improved in next steps... ;)

**Pull request merged**